pages/views/sitemap.njk

```diff
@@ -0,0 +1,17 @@
+---
+permalink: "/sitemap.xml"
+eleventyExcludeFromCollections: true
+---
+
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+{% for page in collections.all %}
+{% if page.data.ignore != true %}
+  <url>
+    <loc>{{ site.url }}{{ page.url.replace(r/\/$/, ".html") if page.data.tags else page.url }}</loc>
+    <lastmod>{{ page.date.toISOString() }}</lastmod>
+    <priority>{{ "1.00" if page.url == "/" else "0.90" }}</priority>
+  </url>
+{% endif %}
+{% endfor %}
+<urlset>
```

Comment:
Major: Incorrect closing tag (opening tag instead of closing one) + empty end-line